Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,21 +1,24 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+import CalculatorDisplay from "./components/DisplayComponents/CalculatorDisplay";
+import NumberButton from "./components/ButtonComponents/NumberButton";
+import ActionButton from "./components/ButtonComponents/ActionButton";
 
 const App = () => {
+  const numArr = ["clear", 7, 8, 9, 4, 5, 6, 1, 2, 3, 0];
+  const signArr = ["÷", "x", "-", "+", "="];
```

Comment:
we will learn more about this today, but you must store data in your react app in the state of your component
```suggestion
  constructor() {
  state: {
    numArr : ["clear", 7, 8, 9, 4, 5, 6, 1, 2, 3, 0],
    signArr : ["÷", "x", "-", "+", "="]
  }
}
```